nixos/modules/services/monitoring/prometheus/default.nix

```diff
@@ -1612,6 +1617,8 @@ in
       '';
     };
 
+    enableAgentMode = mkEnableOption (lib.mdDoc "Enables agent mode.");
```

Comment:
`mkEnableOption` starts the phrase with `Whether to enable` and ends with a period already, so the only think you need is `agent mode` here.